src/inference/optimize.js

```diff
@@ -26,7 +26,25 @@ module.exports = function(env) {
     EUBO: require('./eubo')(env)
   };
 
-  function Optimize(s, k, a, wpplFn, options) {
+  function Optimize(s, k, a, fnOrOptions, maybeOptions) {
+    var wpplFn, options;
```

Comment:
I agree this is inconsistent -- thanks! I think the fix should be to change `Infer` back to using `wpplFn` rather than `model`, since that's what's used in lots of other places, including similar helpers in `header.wppl`. I've pushed a commit that does this, does that seem OK to you?